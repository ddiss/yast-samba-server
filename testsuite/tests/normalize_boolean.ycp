{

// test for normalize_boolean
// $Id$

    include "testsuite.ycp";
    
    include "samba-server/routines.ycp";

    DUMP( normalize_boolean( "yes", nil ) );    
    DUMP( normalize_boolean( "yEs", nil ) );    
    DUMP( normalize_boolean( "1", nil ) );    
    DUMP( normalize_boolean( "true", nil ) );    
    DUMP( normalize_boolean( "TRue", nil ) );    
    DUMP( normalize_boolean( "no", nil ) );    
    DUMP( normalize_boolean( "0", nil ) );    
    DUMP( normalize_boolean( "FALSE", nil ) );    
    DUMP( normalize_boolean( nil, true ) );    
}
