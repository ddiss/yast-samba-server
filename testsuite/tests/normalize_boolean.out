Dump	true
Dump	true
Dump	true
Dump	true
Dump	true
Dump	false
Dump	false
Dump	false
Dump	true
